src/main/java/net/dv8tion/jda/internal/entities/GuildImpl.java

```diff
@@ -1365,6 +1365,20 @@ public long getIdLong()
         return ban0(userId, delDays, reason);
     }
 
+    @Nonnull
+    @Override
+    public AuditableRestAction<Void> ban(long userId, int delDays, String reason)
```

Comment:
why not just use the string overload with `Long.toUnsignedString` instead of copying the whole impl?

Reply:
yep, you're right